HousingSearchApi.Tests/V1/E2ETests/Stories/GetAssetRelationshipsStories.cs

```diff
@@ -0,0 +1,56 @@
+using HousingSearchApi.Tests.V1.E2ETests.Fixtures;
+using HousingSearchApi.Tests.V1.E2ETests.Steps;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace HousingSearchApi.Tests.V1.E2ETests.Stories
+{
+    [Story(
+        AsA = "Service",
+        IWant = "The Asset Search Endpoint to return related assets",
+        SoThat = "it is possible to construct an asset tree")]
+    [Collection("ElasticSearch collection")]
+    public class GetAssetRelationshipsStories
+    {
+        private readonly MockWebApplicationFactory<Startup> _factory;
+        private readonly AssetFixture _assetsFixture;
+        private readonly GetAssetRelationshipsSteps _steps;
+
+        public GetAssetRelationshipsStories(MockWebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+            var httpClient = factory.CreateClient();
+            var elasticClient = factory.ElasticSearchClient;
+
+            _steps = new GetAssetRelationshipsSteps(httpClient);
+            _assetsFixture = new AssetFixture(elasticClient, httpClient);
+        }
+
+        [Fact]
+        public void ServiceReturnsBadResult()
+        {
+            this.Given(g => _assetsFixture.GivenAnAssetIndexExists())
+                .When(w => _steps.WhenRequestDoesNotContainSearchString())
+                .Then(t => _steps.ThenTheLastRequestShouldBeBadRequestResult(default))
+                .BDDfy();
+        }
+
+        [Fact]
+        public void ServiceReturnsOkResult()
+        {
+            this.Given(g => _assetsFixture.GivenAnAssetIndexExists())
+                .When(w => _steps.WhenRequestContainsSearchString())
+                .Then(t => _steps.ThenTheLastRequestShouldBe200())
+                .BDDfy();
+        }
```

Comment:
Should there be a test for a no content response (or no results)?

Reply:
Removed nc response